packages/react-tinacms-inline/src/styles/focus-ring.tsx

```diff
@@ -15,21 +15,70 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 */
+import * as React from 'react'
 import styled, { css } from 'styled-components'
 import { BlocksEmptyState } from '../blocks/inline-field-blocks'
+import { useInlineForm } from '..'
 
-export interface FocusRingStyleProps {
+export interface FocusRingOptions {
   offset?: number | { x: number; y: number }
   borderRadius?: number
+  nestedFocus?: boolean
 }
 
-interface FocusRingProps extends FocusRingStyleProps {
+export interface StyledFocusRingProps {
+  offset?: number | { x: number; y: number }
+  borderRadius?: number
   active: boolean
   disableHover?: boolean
   disableChildren?: boolean
 }
 
-export const FocusRing = styled.div<FocusRingProps>(p => {
+export interface FocusRingProps {
+  name?: string
+  children?: any
+  options?: boolean | FocusRingOptions
```

Comment:
My gut says that this `options` shouldn't be on `FocusRingProps`, but that it should be `FocusRingProps extends FocusRingOptions` but I know this situation is kind of weird so that's fine I guess

Reply:
Right, I think it was using the 'extends' syntax before I made my changes. I was just confused so I did what made sense to me at the time but I figured I'd need help improving the interfaces.